crypto/x509/x509_req.c

```diff
@@ -139,7 +139,7 @@ static STACK_OF(X509_EXTENSION) *get_extensions_by_nid(const X509_REQ *req,
                       ASN1_ITEM_rptr(X509_EXTENSIONS));
 }
 
-STACK_OF(X509_EXTENSION) *X509_REQ_get_extensions(const X509_REQ *req)
+STACK_OF(X509_EXTENSION) *X509_REQ_get_extensions(TYPEQOSSL_CONST_FUTURE X509_REQ *req)
```

Comment:
Why not just OSSL_CONST_FUTURE ?

Reply:
Argh, this was a search&replace mistake, fixed.

Reply:
X969

Reply:
For better alignment with potentially further similar such macros, changed to `OSSL_FUTURE_CONST`.